code/controllers/subsystem/job.dm

```diff
@@ -162,8 +163,8 @@ SUBSYSTEM_DEF(job)
 
 /datum/controller/subsystem/job/proc/ResetOccupations()
 	JobDebug("Occupations reset.")
-	for(var/i in GLOB.new_player_list)
-		var/mob/dead/new_player/player = i
+	for(var/i in GLOB.new_player_list)
```

Comment:
Could you try to figure out whats going on here? Did the whitespace get changed or smth?